en/docs/guides/access-delegation/configure-jwt-grant.md

```diff
@@ -111,34 +111,38 @@ You have successfully added the identity provider. Next, configure the applicati
     - **iat:** This is the epoch of the token issuance time (e.g., 1575024942).
     - **jit:** This is the epoch of the token expiry time (e.g., 1575107914).
 
-    ```tab="Example"
-    {
-    "alg": "RS256",
-    "type": "JWT"
-    }
-
-    {
-    "iss": "NCkZofT51NVKK2UuQSvxPJhQOWwa",
-    "sub": "NCkZofT51NVKK2UuQSvxPJhQOWwa", 
-    "aud": "https://localhost:9443/oauth2/token"
-    "iat": "1575024942",
-    "jit": "1575107914"
-    }
-
-    {
-    <Signature>
-    }
-    ```
+	!!! example "Example"
```

Comment:
since a tab is not actually required here, let's remove the example box and just have it as a code block